contracts/precompiles/lib/GoogleProtobufAny.sol

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+import "./ProtoBufRuntime.sol";
+
+library GoogleProtobufAny {
```

Comment:
why is this needed?

Reply:
Again, something I've exported. It allows us to use a few functions, but it is something we can remove.